### PR TITLE
Reformatted files

### PR DESCRIPTION
**Pull request by buttercrab**
- Add trailing lines
- Add more blank lines
- Add badges for README.md
- Add document in README.md

### CODE REVIEW

**Review by sboh1214 — APPROVED** [review]
Maybe too many badges :)

**Comment by buttercrab** [comment]
> Maybe too many badges :)

There are a lot of actions

File: README.md [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+- fast UDP connection
+- secure TCP connection using TLS
+- transfers data using protobuf
+- pinging for heartbeat
+- reconnection when internet is temporary disabled (in client)
+- functional API that can be called in server & client
+- connection to credential server for authentication
+- version matching for compatability
+- beautiful logging support
```
Comment (utilForever) [inline README.md:24]:
```suggestion
- Fast UDP connection
- Secure TCP connection using TLS
- Transfers data using protobuf
- Pinging for heartbeat
- Reconnection when internet is temporary disabled (in client)
- Functional API that can be called in server & client
- Connection to credential server for authentication
- Version matching for compatability
- Beautiful logging support
```

**Review by utilForever — APPROVED** [review]
I suggest change the uppercase for first letter. 😉

**Review by utilForever — APPROVED** [review]
LGTM now. 🕊️

**Review by sboh1214 — APPROVED** [review]
굳이 shields.io를 써야 하나?

**Pull request merged**